looprpc/client.swagger.json

```diff
@@ -25,6 +21,12 @@
             "schema": {
               "$ref": "#/definitions/looprpcSwapResponse"
             }
+          },
```

Comment:
Hm I think I need to downgrade protoc, @guggero loop is still on the older version, right?

Reply:
We updated now, I think.